var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -31,23 +31,44 @@ class Zfp(Package):
        target error bounds or bit rates.  Although bit-for-bit lossless
        compression is not always possible, zfp is usually accurate to
        within machine epsilon in near-lossless mode, and is often orders
-       of magnitude more accurate than other lossy compressors.
-
+       of magnitude more accurate than other lossy compressors. Versions
+       of zfp 0.5.1 or newer also support compression of integer data.
     """
 
     homepage = "http://computation.llnl.gov/projects/floating-point-compression"
-    url      = "http://computation.llnl.gov/projects/floating-point-compression/download/zfp-0.5.0.tar.gz"
+    url      = "http://computation.llnl.gov/projects/floating-point-compression/download/zfp-0.5.1.tar.gz"
+    list_url = "http://computation.llnl.gov/projects/floating-point-compression/download"
```

Comment:
i don't think you need that, and neither `list_depth` here as it points to the same location as the main `url`.